apps/qdirstat/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "1.8.1-ls82"
+services:
+    qdirstat:
+        container_name: qdirstat
+        restart: unless-stopped
+        ports:
+            - ${APP_PORT}:3000
+        volumes:
+            - ${APP_DATA_DIR}/data/qdirstat/config
+            - ${APP_DATA_DIR}/data/qdirstat/app/data
```

Comment:
I think this app would be more useful if we could see the whole system tree from `/` what do you think?

Reply:
You're right, I think that would solve the problem I alluded to [here](https://github.com/meienberger/runtipi-appstore/pull/1161#issuecomment-1710990364). Feel free to make the change, or ping me if you would like me to!